apps/ops/serializers/variable.py

```diff
@@ -28,6 +28,7 @@ class VariableSerializer(CommonBulkModelSerializer):
     type = LabeledChoiceField(
         choices=FieldType.choices, default=FieldType.text, label=_("Variable Type")
     )
+    default_value = serializers.CharField(max_length=2048, label=_('Default Value'), required=False, allow_blank=True)
     extra_args = serializers.CharField(
         max_length=1024, label=_("ExtraVars"), required=False, allow_blank=True,
         help_text=_(
```

Comment:
There isn't any information about what specific differences you're asking for or how they relate to the code. Please clarify your question with more detailed instructions so I can assist better.